tasks/wait-for-image-task.yaml

```diff
@@ -46,6 +46,25 @@ spec:
         exit 1
       fi
 
+      # Output
       echo -n "${infos[0]}" | tee "$(results.IMAGE_DIGEST.path)"
       echo -n "${infos[1]}" | tee "$(results.GIT_REF.path)"
       echo -n "${infos[2]}" | tee "$(results.GIT_REPO.path)"
+
+  - name: wait-for-attestation
+    image: registry.redhat.io/rhtas/cosign-rhel9:1.2.0-1744791100@sha256:cb53dcc3bc912dd7f12147f33af1b435eae5ff4ab83b85c0277b4004b20a0248
+    script: |
+      #!/usr/bin/env bash
+
+      set -euo pipefail
+
+      # Wait for the attestation OCI to appear.
```

Comment:
[nit] I think the comment is redundant given that the next `echo` line communicates the same.
```suggestion
```